install-worker.sh

```diff
@@ -137,6 +137,7 @@ sudo rm -rf \
 # Clean up files to reduce confusion during debug
 sudo rm -rf \
     /home/ec2-user/.ssh/authorized_keys \
+    /root/.ssh/authorized_keys \
```

Comment:
just info: this file needed to show nice message if someone try to run `ssh root@IP`
this file will be regenerated during the first instance start by cloud-init.